docs/source/package_reference/models.mdx

```diff
@@ -0,0 +1,17 @@
+# Models
+
```

Comment:
I would split into sections to separate:
- LightevalModel (our abc iirc)
- BaseModel/DeltaModel/AdapterModel (transformers/accelerate)
- InferenceEndpoints (allows to launch providers), maybe grouped with ModelClient 
- Nanotron
- VLLM

Reply:
Done: [b2d82e3](https://github.com/huggingface/lighteval/pull/413/commits/b2d82e3e68055ffb9a4c6c94933c08479481481d)